### PR TITLE
[T4A5][W11-B4]Ong Heng Le 

### PR DESCRIPTION
**Pull request by initialshl**

### CODE REVIEW

**Comment by initialshl** [comment]
Ready for review

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents a Tagging associated with the person and tag operation.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:6]: :+1:  for the right comment!

File: src/seedu/addressbook/data/AddressBook.java [hunk 2]
```diff
@@ -22,13 +24,15 @@
...
     private final UniqueTagList allTags; // can contain tags not attached to any person
+    private LinkedList<Tagging> allTaggings;
 
     /**
      * Creates an empty address book.
      */
     public AddressBook() {
         allPersons = new UniquePersonList();
         allTags = new UniqueTagList();
+        allTaggings = new LinkedList<Tagging>();
```
Comment (okkhoy) [inline src/seedu/addressbook/data/AddressBook.java:35]: also you need to print the tags when the session ends!

Comment (okkhoy) [inline src/seedu/addressbook/data/AddressBook.java:35]: also you need to print the tags when the session ends!

**Comment by okkhoy** [comment]
@initialshl 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**